.github/workflows/multi-py-deb-workflow.yml

```
@@ -40,6 +40,10 @@ on:
         required: false
         type: string
         default: 'false'
+      install_test_distros:
+        required: false
+        type: string
+        default: '["20.04", "22.04", "24.04"]'
```

To keep backwards compatibility it should be
```suggestion
        default: '["20.04", "22.04"]'
```